easybuild/easyblocks/m/mono.py

```diff
@@ -140,14 +138,15 @@ def configure_step(self):
             if self.cfg['parallel']:
                 par = "-j %s" % self.cfg['parallel']
 
-            config_cmd = "%s ./configure --prefix=%s %s" % (self.cfg['preconfigopts'], tmp_mono_path, self.cfg['configopts'])
+            config_cmd = "%s ./configure --prefix=%s %s" % (
+                self.cfg['preconfigopts'], tmp_mono_path, self.cfg['configopts'])
```

Comment:
Why not make it consistent with how `build_cmd` is constructed below:

```suggestion
            config_cmd = ' '.join([
                self.cfg['preconfigopts'],
                "./configure",
                "--prefix=" + tmp_mono_path,
                self.cfg['configopts'],
            ])